README.md

```diff
@@ -1,3 +1,4 @@
 # Arbiter
 
-[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/RazoftOSS/Arbiter?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
\ No newline at end of file
+[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/RazoftOSS/Arbiter?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![Build Status](http://cpssd1-web.computing.dcu.ie/buildStatus/icon?job=Arbiter Master)](http://cpssd1-web.computing.dcu.ie/job/Arbiter%20Master/)
```

Comment:
Should there be a space between `Arbiter` and `Master`? I know its auto generated by Jenkins but it should be escaped with `%20`